src/adoptawidget/index.md

```diff
@@ -4,6 +4,8 @@ layout: default
 show_banner: False
 ---
 
+<?code-excerpt path-base="adopt_a_widget"?>
```

Comment:
I'm wondering if this page should be code excerpts at all? @domesticmouse They're all technically just comments...

Reply:
I'm curious why we are excerpting comments, there is no way to verify if the code shown is correct.

Reply:
I thought originally there may have been _actual_ code on the page (it was listed on the sheet that outlined which pages needed excerpts) and maybe we'd reintegrate. I can scrap that entire project since it's just comments right now.

Reply:
Cool!